pkgs/tools/audio/piper/default.nix

```
@@ -25,6 +25,10 @@ stdenv.mkDerivation {
 
   sourceRoot = "source/src/cpp";
 
+  patches = [
+    ./breaking-change-onnxruntime.patch
```

```suggestion
    ./fix-compilation-with-newer-onnxruntime.patch
```
or something similar